java/server/src/org/openqa/selenium/redis/RedisClientHelper.java

```diff
@@ -0,0 +1,42 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.redis;
+
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.RedisURI;
+import io.lettuce.core.api.StatefulRedisConnection;
+
+import java.net.URI;
+
+public class RedisClientHelper {
```

Comment:
From the point of view of our system, this _is_ the redis client, or (perhaps) the connection. Name the class after what it does in our system and why it's special. Names like `Helper`, `Utility`, `Support` and `Impl` all fail to clearly explain the actual role the class is fulfilling, and so should be avoided.